src/SPC/builder/extension/dom.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPC\builder\extension;
+
+use SPC\builder\Extension;
+use SPC\exception\RuntimeException;
+use SPC\store\FileSystem;
+use SPC\util\CustomExt;
+
+#[CustomExt('dom')]
+class dom extends Extension
+{
+    /**
+     * @throws RuntimeException
+     */
+    public function getUnixConfigureArg(bool $shared = false): string
+    {
+        $arg = '--enable-dom' . ($shared ? '=shared' : '');
+        $arg .= ' --with-libxml="' . BUILD_ROOT_PATH . '"';
```

Comment:
We could change dom's `ext-depends-windows` to `ext-depends` and remove this line.

Reply:
(also, passing this option multiple times is harmless. our xml.php will set it multiple times anyhow).